corehq/apps/reports/generic.py

```diff
@@ -294,19 +298,29 @@ def timezone(self):
     @property
     @memoized
     def template_base(self):
+        if self.use_bootstrap5:
+            return get_bootstrap5_path(self.base_template)
         return self.base_template
 
     @property
     @memoized
     def template_async_base(self):
+        if self.use_bootstrap5:
+            default_path = "reports/async/bootstrap5/default.html"
+        else:
+            default_path = "reports/async/bootstrap3/default.html"
         if self.asynchronous:
-            return self.base_template_async or "reports/async/bootstrap3/default.html"
+            return self.base_template_async or default_path
```

Comment:
should this use `get_bootstrap5_path`?

Reply:
updated here https://github.com/dimagi/commcare-hq/pull/35648/commits/b564d7435b27abc9cb8d1bcfb7075cdb4f63e764